content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -0,0 +1,254 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Ephemeral Volumes
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+This document describes the current state of _ephemeral volumes_ in Kubernetes. Familiarity with [volumes](/docs/concepts/storage/volumes/) is suggested.
+
+<!-- body -->
+
+## Introduction
+
+Some application need additional storage but don't care whether that
+data is stored persistently across restarts. For example, caching
+services are often limited by memory size and can move infrequently
+used data into storage that is slower than memory with little impact
+on overall performance.
+
+Other applications expect some read-only input data to be present in
+files, like configuration data or secret keys.
+
+_Ephemeral volumes_ are designed for these use cases. Because volumes
+are created anew for each pod, pods can be stopped and restarted
+without being limited to where some persistent volume is available.
+
+Ephemeral volumes are specified _inline_ in the pod spec, which
+simplifies application deployment and management.
+
+Kubernetes supports several different kinds of ephemeral volumes for
+different purposes:
+- [emptyDir]((/docs/concepts/volumes/#emptydir): a directory on the root disk or
```

Comment:
Does the backing storage `emptyDir` have to live on the node's root filesystem? (I don't believe so).

I'd explain it as “an amount of node-local ephemeral storage, empty at Pod startup”

Reply:
> The kubelet has a base directory for other locally stored data (`/var/lib/kubelet` by default).
> 
> Typically, `/var/lib/kubelet` is on the system root filesystem, and the kubelet is designed with that layout in mind.

(adapted from https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage)

I don't think “root disk” is correct. First because the root filesystem isn't always backed by durable storage, and second because `/var/lib/kubelet` doesn't have to live on that filesystem.

Reply:
"root disk" is a simplification. How about "with storage provided locally from the kubelet base directory (usually the root disk) or from memory"?

I'm also not sure about "provided locally from" - perhaps "coming locally from"?

Reply:
I've switched to "with storage coming locally from the kubelet base directory (usually the root disk) or RAM"